### DISCUSSION TITLE
Growth of variants

### DISCUSSION THREAD

**Discussion by adamkucharski** [WHO-Collaboratory/collaboratory-epipipeline-community#23] [Ideas]
- Pipeline for GISAID (and other) databases, (multinomial) logistic growth models
- R package to analyze the growth advantage of new (SARS-CoV-2) variants from genomic surveillance data

WHO Collaboratory Hackathon Idea.

**Comment by adamkucharski** [comment]
This was used quite a lot for COVID so might be a good starting point: https://github.com/tomwenseleers/LineageExplorer

**Reply by calthaus** [reply]
The [Gerstung Lab](https://github.com/gerstung-lab) also has some material (albeit in Python):
- https://github.com/gerstung-lab/genomicsurveillance
- https://github.com/gerstung-lab/SARS-CoV-2-International

@lborcard and @malare from the University of Bern have worked on a Shiny app to track SARS-CoV-2 variants, which could be included in a package. @finlaycampbell might also be interested to contribute to such a package development.

**Reply by finlaycampbell** [reply]
Yes happy to be involved/kept in the loop with this! I did quite a lot of internal work on this but we never got round to publishing the code even though it is fairly well-documented/packaged. I used the `outbreakinfo` package to pull GISAID data, though also had the option to import the GISAID data from a locally downloaded file.

**Reply by calthaus** [reply]
@jabacker from RIVM has also worked on this and could give valuable input. I will try to set up a meeting with all of us in the coming weeks to discuss how we move forward.

**Reply by adamkucharski** [reply]
Oliver Johnson at Bristol has also been doing regular updates on COVID variants using some MLE-based approaches (e.g. https://bristoliver.substack.com/p/covid-graphs-update) so expect he'll have some useful input on common obstacles/bottlenecks in producing quick early insights – he said he'd be interested in being looped in.

**Comment by adamkucharski** [comment]
Talking with Moritz Kraemer, he'd also be interested in discussing – they're working on pipelines for genomic analysis with [GRAPEVNE](https://grapevne.readthedocs.io/en/latest/) so could be an opportunity to build module here.
